<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1474</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1181</width>
      <height>781</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="settings_widget">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QPushButton" name="start_measurements_button">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>160</y>
        <width>201</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Start measurements</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sampling_rate_line_edit">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>100</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="sampling_rate_prompt_label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>701</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Enter frequency of BME280 measurements (in seconds)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="current_sampling_rate_label">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>330</y>
        <width>701</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="sampling_rate_info_label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>306</y>
        <width>861</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sampling rate info will show up here</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>List files</string>
     </attribute>
     <widget class="QPushButton" name="list_files_button">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>70</y>
        <width>281</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string>List files</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="list_files_text_browser">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>260</y>
        <width>491</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="last_measurements_widget">
     <attribute name="title">
      <string>Last measurements</string>
     </attribute>
     <widget class="QLabel" name="pressure_label">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>520</y>
        <width>241</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="humidity_label">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>310</y>
        <width>241</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="temperature_label">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>130</y>
        <width>191</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>145</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Last temperature</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>320</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Last humidity</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>530</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Last pressure</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Last update</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="last_update_label">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>40</y>
        <width>291</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="plot_widget">
     <attribute name="title">
      <string>Plot</string>
     </attribute>
     <widget class="QPushButton" name="plot_button">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>580</y>
        <width>221</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
     <widget class="PlotWidget" name="temperature_plot" native="true">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>401</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>-130</x>
        <y>10</y>
        <width>891</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Temperature plot</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="PlotWidget" name="humidity_plot" native="true">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>70</y>
        <width>411</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>10</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Humidity plot [%]</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="file_name_line_edit">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>510</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="PlotWidget" name="pressure_plot" native="true">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>430</y>
        <width>401</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>380</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pressure plot</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>480</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter file name to plot</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>150</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperature [deg C]</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>510</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pressure [hPa]</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>150</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Humidity [%]</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>340</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Time from starting measurements [s]</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>710</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Time from starting measurements [s]</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>340</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Time from starting measurements [s]</string>
      </property>
     </widget>
     <widget class="QLabel" name="datetime_info_label">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>430</y>
        <width>411</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Information about starting measurements will show up here</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1474</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
